src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -101,6 +101,8 @@ public bool HasDescription
             get { return (!string.IsNullOrEmpty(Model.Description)); }
         }
 
+        public NodeCategoryViewModel Category { get; set; }
+
```

Comment:
Is it possible to make `Category` a `private set`? I think we can update `NodeSearchElementViewModel` constructor to extract the information out of the `NodeSearchElement` it receives. Does that make sense?